task02.rb

```diff
@@ -0,0 +1,28 @@
+#2. For a given array that contains some strings output the content of those strings separated with whitespace between them, having capitalized first word and ending with ".", e.g. [[" thIs ", "Is"], nil, " My  ", ["   first"], "", :program] should output "This is my first program."
+
+def task02(arr)
+  if arr.class != Array
+    raise ArgumentError, "The input type is invalid, expect an array"
+  end
+  if arr.empty?
+    raise ArgumentError, "The input is empty."
+  end
+
+  format_string = arr.flatten.join(" ").strip!
```

Comment:
@MaraEfimova Be careful what the bang method does and returns. This leads to unexpected behavior (e.g. `['me']` throwing an exception).